lua/framework/physics/lua_material.cc

```diff
@@ -96,7 +96,7 @@ MatAddProperty(lua_State* L)
   auto cur_material = opensn::GetStackItemPtr(opensn::material_stack, material_index, fname);
 
   auto provided_name = LuaArgOptional<std::string>(
-    L, 3, std::string("Property ") + std::to_string(cur_material->properties_.size()));
+    L, 3, std::string("Property ") + std::to_string(cur_material->properties.size()));
 
   // Process property
   using MatProperty = PropertyType;
```

Comment:
Could you remove this alias while you are modifying this code?

Reply:
Done.